### DISCUSSION TITLE
Multi-tenant support

### DISCUSSION THREAD

**Discussion by jaekim0908** [vendurehq/vendure#1164] [General]
Hi,

I'd like to use vendura as the backend e-commerce platform for my customers and would like to support multi-tenancy. Is there any plug-ins to support this? If not do you guys provide any design consulting or development work?

Thanks,
Jae.

**Comment by michaelbromley** [comment]
Hi!

> Is there any plug-ins to support this?

No, not at the moment. It is a possibility for the future but for now, it is possible to implement multi-tenancy with some custom work based on the Channels feature.

> If not do you guys provide any design consulting or development work?

Consulting yes, see https://www.vendure.io/support/. For development, please see our partners list, who all have experience building production Vendure projects: https://www.vendure.io/partners/

**Comment by jaekim0908** [comment]
Thank you for your reply.

Could you share more info regarding using channels feature to support multi
tenancy?

Thanks,
Jae.

**Reply by michaelbromley** [reply]
There is a small amount of docs here: https://www.vendure.io/docs/developer-guide/channels/#multi-tenant-marketplace-support, not much detail yet I'm afraid.

**Comment by michaelbromley** [comment]
Tracking this here: https://github.com/vendure-ecommerce/vendure/issues/1329

Please add further discussion to that issue.
